packages/ui/components/form-core/src/registration/FormRegisteringMixin.js

```diff
@@ -36,6 +36,7 @@ const FormRegisteringMixinImplementation = superclass =>
         new CustomEvent('form-element-register', {
           detail: { element: this },
           bubbles: true,
+          composed: this.formElementRegisterComposed ? true : false,
```

Comment:
```suggestion
          composed: this.allowCrossRootRegistration,
```

Reply:
Can you add this to constructor?

```js
/**
 * To encourage accessibility best practices, `form-element-register` events
 * do not pierce to shadow roots. This forces the developer to create form groups and fieldsets that automatically can create accessible relationships in the same dom tree.
 If you know what you're doing, it is possible to nest FormControls
 inside shadow dom. See https://lion-web.netlify.app/fundamentals/rationales/accessibility/#shadow-roots-and-accessibility
 */
this.allowCrossRootRegistration = false;
```

Reply:
And the same field should be added to the mixin types: https://github.com/ing-bank/lion/blob/master/packages/ui/components/form-core/types/registration/FormRegisteringMixinTypes.ts

Reply:
And can you add this to properties?

```js
allowCrossRootRegistration: {type: Boolean, attribute: allow-cross-root-registration}
```